### PR TITLE
Regions: position publish region after sidebar

### PR DESCRIPTION
**Pull request by ellatrix**
## Description

Currently the button for opening the publish area is positioned right between the block and the block inspector/sidebar. This is a bit of an awkward place for it. I moved the button after the inspector instead. This makes visual sense as well, as the publish area slides in from the left (seemingly positioned after the sidebar).

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR. .

### CODE REVIEW

**Comment by gziolo** [comment]
I tested it and it feels much better when the publish flow triggers after the document/block sidebar 👍 Thank you for working on it. 

It looks like some e2e tests need to get updated to align with the changes proposed.

File: packages/edit-post/src/components/editor-regions/index.js [hunk 1]
```diff
@@ -33,25 +33,25 @@ function EditorRegions( { footer, header, sidebar, content, publish, className }
...
 				</div>
-				{ !! publish && (
+				{ !! sidebar && (
 					<div
-						className="edit-post-editor-regions__publish"
+						className="edit-post-editor-regions__sidebar"
 						role="region"
-						/* translators: accessibility text for the publish landmark region. */
-						aria-label={ __( 'Editor publish' ) }
+						aria-label={ 'Editor settings' }
```
Comment (MarcoZehe) [inline packages/edit-post/src/components/editor-regions/index.js]: For consistency, you might want to add a comment for translators here as well.

**Review by MarcoZehe — COMMENTED** [review]
Thanks for working on this, this will also make screen reader content flow much more logical.

File: packages/edit-post/src/components/editor-regions/index.js (hunk 1 above)
Comment (gziolo) [reply packages/edit-post/src/components/editor-regions/index.js]: Good catch, it should be wrapped with the translate helper function as well 👍

**Comment by ellatrix** [comment]
Actually I think the publish region should remain in its current place because it is above the sidebar. See #19625.

File: packages/edit-post/src/components/editor-regions/index.js (hunk 1 above)
Comment (ellatrix) [reply packages/edit-post/src/components/editor-regions/index.js]: Ah, looks like it wasn't there before.

**Comment by gziolo** [comment]
🎉 Muchas gracias!

**Pull request merged**